activities/models/post_attachment.py

```diff
@@ -122,6 +122,11 @@ def to_ap(self):
             "mediaType": self.mimetype,
             "blurhash": self.blurhash,
         }
+        if self.is_image() and self.focal_x and self.focal_y:
+            ap["type"] = "Image"
+            ap["mediaType"] = self.mimetype
```

Comment:
This seems redundant?

Reply:
mediatype, you're right. for type = "Image", not sure. Mastodon sets and sends focalpoint on type = Image attachments, and it doesn't seem right to try to set that on type = Document attachments instead. This is one of those cases where end-to-end interop testing is not simple - it turns out none of the clients I use even support defining a focal point, so I can't send a test message..

Reply:
Yeah, it was just the double mediaType assignment I was calling out!